hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenUpdateNftsHandler.java

```diff
@@ -151,4 +171,37 @@ private void validateSemantics(
         }
         validateTrue(op.serialNumbers().size() <= tokensConfig.nftsMaxBatchSizeUpdate(), BATCH_SIZE_LIMIT_EXCEEDED);
     }
+
+    /**
+     * Creates a threshold key with threshold 1 with the given keys.
+     * @param keysRequired keys required
+     * @return threshold key with threshold 1
+     */
+    private Key oneOf(@NonNull final Key... keysRequired) {
+        return Key.newBuilder()
+                .thresholdKey(ThresholdKey.newBuilder()
+                        .keys(new KeyList(Arrays.asList(keysRequired)))
+                        .threshold(1)
+                        .build())
+                .build();
```

Comment:
I think this works? It does feel a little odd to create a new key for each qualifying update 🤔

Reply:
Yeah, TokenUpdateHandler makes use of this currently.